tests/std/tests/P0122R7_span/test.cpp

```diff
@@ -63,50 +63,18 @@ static_assert(is_same_v<span<const int, 3>::const_reference, const int&>);
 
 // The iterators are thoroughly tested by P0896R4_ranges_range_machinery...

```

Comment:
This ellipsis should be changed to a period, as the “... except for” comment below is being removed.

Reply:
Done